platform_design/lib/widgets.dart

```diff
@@ -300,7 +302,7 @@ void showChoices(BuildContext context, List<String> choices) {
                       value: index,
                       groupValue: selectedRadio,
                       // ignore: avoid_types_on_closure_parameters
-                      onChanged: (int value) {
+                      onChanged: (int? value) {
```

Comment:
You should be able to remove this type specifier entirely (and the comment on line 304), if you set the `RadioListTile` to be a.`RadioListTile<int?>`.